lib/bamboo/adapters/smtp_adapter.ex

```diff
@@ -161,6 +168,45 @@ defmodule Bamboo.SMTPAdapter do
     |> add_smtp_line(text_body)
   end
 
+  defp add_multipart_mixed_split(body, delimiter), do: "#{body}\r\n--#{delimiter}\r\n"
+
+  defp add_attachment_header(body, attachment) do
+    << random :: size(32) >> = :crypto.strong_rand_bytes(4)
+    "#{body}Content-Type: attachment.content_type; name=\"#{attachment.filename}\"\r\n" <>
```

Comment:
Sorry, my english fails here, do you mean that we should check if attachment.content_type is set before using? If so, we won't have to do that here. Bamboo.Attachment already sets content_type to "application/octet-stream" if not set on creation. If you meant something else, please let me now and I'll fix it. Thanks for the suggestions! :)